file2_name.csv

```diff
@@ -0,0 +1,65 @@
+5b17df39811d4a000470c512,US,Null,Jupiter Inlet Beach Colony,26.9374,-80.0711,,
+5b17f83041563200049f9520,SK,Null,Vrchovany,48.9633,18.6101,,
+5b17dcf2811d4a000470bb84,AU,Null,Bonny Hills,-31.5897,152.84,,
+5b17def2811d4a000470c3fe,US,Null,Allegheny Center,40.4505,-79.9978,,
+5b17f87b41563200049f9656,ES,Null,L'estany,41.8692,2.11198,,
+5b17f4a241563200049f8650,US,Null,Sunnyview,44.3697,-96.7904,,
+5b17e077811d4a000470ca12,US,Null,Alleghany,39.4716,-120.843,,
+5b0d403084d57c0004cba7aa,US,IA,Sunnybrook,42.4555,-96.3356,704,
+5b17f50e41563200049f880e,CH,Null,Martigny-bourg,46.0984,7.07245,,
+5b17dcf5811d4a000470bb8e,AU,Null,Bonny Hills,-31.5897,152.838,,
+5b17f4de41563200049f8742,CH,Null,Martigny-ville,46.101,7.07256,,
+5b17f87641563200049f9642,ES,Null,Cerdanyola Del Valls,41.4839,2.16036,,
+5b17df23811d4a000470c4b7,US,Null,Washington Colony (historical),48.7368,-122.485,,
+5b17f62c41563200049f8cd3,ES,Null,Ontinyent,38.826,-0.597124,,
+5b17f6c541563200049f8f57,RU,Null,Listvennyy,56.8405,60.6198,,
+5b17e021811d4a000470c8f2,US,Null,Jamestown Colony,31.3202,-85.8326,,
+5b17df70811d4a000470c602,CA,Null,Stony Rapids,59.2494,-105.835,,
+5b17de90811d4a000470c258,US,Null,Grand Canyon Village,36.0512,-112.15,,
+5b17e025811d4a000470c902,US,Null,Bryce Canyon City,37.6736,-112.157,,
+5b17df6d811d4a000470c5f5,US,Null,Denny Creek,47.4442,-121.426,,
+5b17df50811d4a000470c577,US,Null,Grand Canyon,36.0549,-112.14,,
+5b17f87541563200049f963c,ES,Null,Son Rapinya,39.5816,2.62273,,
+5b17f87541563200049f963d,ES,Null,Son Rapinya,39.5738,2.6108,,
+5b0d437e84d57c0004cbab13,US,IA,Ankeny,41.7608,-93.5809,,
+5b17deb2811d4a000470c2ec,US,Null,Bryce Canyon City,37.6863,-112.138,,
+5b17f8a341563200049f96f4,UA,Null,Poznyaki,50.4023,30.6338,,
+5b17df61811d4a000470c5c1,US,Null,Canyon Junction,44.7945,-110.432,,
+5b17f5ed41563200049f8bc1,FR,Null,Chaligny,48.6239,6.08515,,
+5b17f87541563200049f963b,ES,Null,Son Rapinya,39.5841,2.62608,,
+5b17def3811d4a000470c400,US,Null,East Allegheny,40.4564,-79.9981,,
+5b17def3811d4a000470c3ff,US,Null,East Allegheny,40.4526,-79.9969,,
+5b17f68741563200049f8e4d,CZ,Null,Trnovany,50.6442,13.8342,,
+5b17f89841563200049f96c4,UA,Null,Novo-poznyaki,50.4033,30.6523,,
+5b17df6b811d4a000470c5ed,CA,Null,Albany Corner,46.2757,-63.615,,
+5b17e028811d4a000470c90f,US,Null,Sunnybank,37.8837,-76.2511,,
+5b17de9a811d4a000470c282,GD,Null,Petit Calivigny,12.0036,-61.7245,,
+5b17dee9811d4a000470c3d5,US,Null,Allegheny West,40.0103,-75.1847,,
+5b17f68141563200049f8e35,CZ,Null,Podboany,50.2279,13.414,,
+5b17df04811d4a000470c431,US,Null,East Allegheny,40.4526,-79.9969,,
+5b17f42441563200049f8434,US,Null,Penny Lynn Mobile Home Park,47.9264,-122.097,,
+5b17f7c541563200049f9364,SK,Null,Lazany,49.0148,18.8088,,
+5b17f61541563200049f8c6d,FR,Null,Champagny-en-vanoise,45.4567,6.69393,,
+5b0d435b84d57c0004cbaaee,US,IA,Ankeny,41.6798,-93.5725,,
+5b17f80241563200049f945f,CZ,Null,Pustiny,50.2291,16.4567,,
+5b17f78441563200049f9255,PL,Null,Mogilany,49.95,19.8833,,

```

Comment:
we should not have this file in our repo

Reply:
I have removed this file from the repo.